libraries/AP_Mount/AP_Mount_Alexmos.cpp

```diff
@@ -162,13 +162,15 @@ void AP_Mount_Alexmos::control_axis(const Vector3f& angle, bool target_in_degree
     alexmos_parameters outgoing_buffer;
     outgoing_buffer.angle_speed.mode_roll = get_control_mode(_state._roll_input_mode);
     outgoing_buffer.angle_speed.mode_pitch = get_control_mode(_state._pitch_input_mode);
-    outgoing_buffer.angle_speed.mode_yaw = get_control_mode(_state._yaw_input_mode);
+//    outgoing_buffer.angle_speed.mode_yaw = get_control_mode(_state._yaw_input_mode);
```

Comment:
Please remove the commented changes in this file

Reply:
Removed commented code